### PR TITLE
fix(wine-staging-git): remove missing dependencies

### PR DESCRIPTION
**Pull request by echometerain**

### CODE REVIEW

**Comment by echometerain** [comment]
last commit removes libgstreamer-plugins-base1.0-dev:i386

**Comment by echometerain** [comment]
turns out there was a duplicate

File: packages/wine-staging-git/wine-staging-git.pacscript [hunk 1]
```diff
@@ -70,24 +70,25 @@ build() {
...
     libdir="${pkgdir}/usr/lib" \
     dlldir="${pkgdir}/usr/lib/wine" \
     install
 
-  install -d -m755 "${pkgdir}/usr/share/fontconfig/conf.default"
-  install -D -m644 "${SRCDIR}/${name}/30-win32-aliases.conf" -t "${pkgdir}/usr/share/fontconfig/conf.avail"
+  sudo install -d -m755 "${pkgdir}/usr/share/fontconfig/conf.default"
+  sudo install -D -m644 "${SRCDIR}/${name}/30-win32-aliases.conf" -t "${pkgdir}/usr/share/fontconfig/conf.avail"
+  # FIXME: this line doesn't work post 3.0.0
   ln -s "../conf.avail/30-win32-aliases.conf" "${pkgdir}/usr/share/fontconfig/conf.default/30-win32-aliases.conf"
```
Comment (Henryws) [inline packages/wine-staging-git/wine-staging-git.pacscript:88]: You can't symlink files in `install`

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/wine-staging-git/wine-staging-git.pacscript [hunk 2]
```diff
@@ -70,24 +70,31 @@ build() {
...
+  sudo install -D -m644 "${SRCDIR}/${name}/wine-binfmt.conf" "${pkgdir}/usr/lib/binfmt.d/wine.conf"
 
-  i686-w64-mingw32-strip --strip-unneeded "${pkgdir}/usr/lib32/wine/i386-windows"/*.dll
-  x86_64-w64-mingw32-strip --strip-unneeded "${pkgdir}/usr/lib/wine/x86_64-windows"/*.dll
+  sudo i686-w64-mingw32-strip --strip-unneeded "${pkgdir}/usr/lib32/wine/i386-windows"/*.dll
+  sudo x86_64-w64-mingw32-strip --strip-unneeded "${pkgdir}/usr/lib/wine/x86_64-windows"/*.dll
+}
+
+postinst() {
+  sudo ln -s "../conf.avail/30-win32-aliases.conf" "${pkgdir}/usr/share/fontconfig/conf.default/30-win32-aliases.conf"
```
Comment (Henryws) [inline packages/wine-staging-git/wine-staging-git.pacscript:95]: postinst is run from `/` so `../` can't exist. Figure out where this file actually installs to

**Review by Henryws — CHANGES_REQUESTED** [review]

**Comment by Henryws** [comment]
Any updates on this PR?

**Comment by Henryws** [comment]
Please don't update the branch if you don't need to.

**Comment by echometerain** [comment]
> Any updates on this PR?

sorry I was busy for the last couple of days

**Comment by echometerain** [comment]
holy moly I think it worked

**Review by Henryws — APPROVED** [review]

**Pull request merged**